test/app/submissions_test.rb

```diff
@@ -43,7 +43,7 @@ def assert_response_status(expected_status)
   end
 
   def test_guests_can_view_submissions
-    Attempt.new(alice, Iteration.new('word-count/file.rb' => 'CODE')).save
+    Attempt.new(alice, Iteration.new('ruby/file.rb' => 'CODE')).save
```

Comment:
X::Track.find with an invalid slug will make a call to the x-api which returns a null track--this should be fine, unless the x-api is down, in which case we're going to have much bigger problems.

I'm aiming to stub as little as possible. I don't want to stub the `X::Language.of` method at all.